lib/modules/filter/query.rb

```diff
@@ -308,11 +308,17 @@ def apply_sort(query, table, column_name, allowed, direction)
       sort_field = @build.build_custom_field(column_name)
       sort_field = table[column_name] if sort_field.blank?
 
-      if direction == :desc
-        sort_field_by = Arel::Nodes::Descending.new(sort_field)
+      if (sort_field.is_a? String)
+        sort_field_by = sort_field
```

Comment:
does this need to be checked for sql injection?

I can't remember but I'm hoping the `params` methods do this in rails. Either that or by using it with `.order` below should ensure safety. Worth checking.